modules/nf-core/basicpy/meta.yml

```diff
@@ -0,0 +1,46 @@
+name: "basicpy"
+description: write your description here
+keywords: illumiation_correction, microscopy, imaging
+  - sort
```

Comment:
Can you add some more keywords?

Reply:
I think I just added the keywords wrong as one line, rather then bullet points! Changed this!